src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -20,5 +21,28 @@ public IObservable<IPullRequestModel> CreatePullRequest(IRepositoryHost host, IS
 
             return host.ModelService.CreatePullRequest(repository, title, body, source, target);
         }
+
+        public string GetPullRequestTemplate(ISimpleRepositoryModel repository)
+        {
+            Extensions.Guard.ArgumentNotNull(repository, nameof(repository));
+
+            var paths = new[]
+            {
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE.md"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE.md"),
```

Comment:
Well we still have to concatenate the strings, and either create a new array to hold those strings, or use LINQ which would involve even more allocations and a far more complex code-path ;)

But it sounds like you do have strong feelings on this, so will change.